layers/+distributions/spacemacs-base/packages.el

```diff
@@ -60,7 +61,8 @@
     (progn
       (spacemacs/set-leader-keys
         "bD" 'spacemacs/ace-kill-this-buffer
-        "wC" 'spacemacs/ace-center-window
+        ;; FIXME: Needs new binding.
+        ;; "wC" 'spacemacs/ace-center-window
```

Comment:
That's OK with me, actually we could remove `spacemacs/ace-center-window` I don't see it to be very useful. Or we move `SPC w C` to `SPC w c` and re-enable this binding.